src/test/java/org/sonarqube/auth/bitbucket/BitbucketSettingsTest.java

```diff
@@ -29,88 +30,97 @@
-    settings.setProperty("sonar.auth.bitbucket.clientId.secured", "id");
-    settings.setProperty("sonar.auth.bitbucket.clientSecret.secured", "secret");
-    settings.setProperty("sonar.auth.bitbucket.loginStrategy", LOGIN_STRATEGY_DEFAULT_VALUE);
-
-    settings.setProperty("sonar.auth.bitbucket.enabled", true);
-    assertThat(underTest.isEnabled()).isTrue();
-
-    settings.setProperty("sonar.auth.bitbucket.enabled", false);
-    assertThat(underTest.isEnabled()).isFalse();
-  }
-
-  @Test
-  public void is_enabled_always_return_false_when_client_id_is_null() {
-    settings.setProperty("sonar.auth.bitbucket.enabled", true);
-    settings.setProperty("sonar.auth.bitbucket.clientId.secured", (String) null);
-    settings.setProperty("sonar.auth.bitbucket.clientSecret.secured", "secret");
-    settings.setProperty("sonar.auth.bitbucket.loginStrategy", LOGIN_STRATEGY_DEFAULT_VALUE);
-
-    assertThat(underTest.isEnabled()).isFalse();
-  }
-
-  @Test
-  public void is_enabled_always_return_false_when_client_secret_is_null() {
-    settings.setProperty("sonar.auth.bitbucket.enabled", true);
-    settings.setProperty("sonar.auth.bitbucket.clientId.secured", "id");
-    settings.setProperty("sonar.auth.bitbucket.clientSecret.secured", (String) null);
-    settings.setProperty("sonar.auth.bitbucket.loginStrategy", LOGIN_STRATEGY_DEFAULT_VALUE);
-
-    assertThat(underTest.isEnabled()).isFalse();
-  }
-
-  @Test
-  public void default_login_strategy_is_unique_login() {
-    assertThat(underTest.loginStrategy()).isEqualTo(BitbucketSettings.LOGIN_STRATEGY_UNIQUE);
-  }
-
-  @Test
-  public void return_client_id() {
-    settings.setProperty("sonar.auth.bitbucket.clientId.secured", "id");
-    assertThat(underTest.clientId()).isEqualTo("id");
-  }
-
-  @Test
-  public void return_client_secret() {
-    settings.setProperty("sonar.auth.bitbucket.clientSecret.secured", "secret");
-    assertThat(underTest.clientSecret()).isEqualTo("secret");
-  }
-
-  @Test
-  public void return_login_strategy() {
-    settings.setProperty("sonar.auth.bitbucket.loginStrategy", LOGIN_STRATEGY_PROVIDER_LOGIN);
-    assertThat(underTest.loginStrategy()).isEqualTo(LOGIN_STRATEGY_PROVIDER_LOGIN);
-  }
-
-  @Test
-  public void allow_users_to_sign_up() {
-    settings.setProperty("sonar.auth.bitbucket.allowUsersToSignUp", "true");
-    assertThat(underTest.allowUsersToSignUp()).isTrue();
-
-    settings.setProperty("sonar.auth.bitbucket.allowUsersToSignUp", "false");
-    assertThat(underTest.allowUsersToSignUp()).isFalse();
-  }
-
-  @Test
-  public void default_apiUrl() {
-    assertThat(underTest.apiURL()).isEqualTo("https://api.bitbucket.org/");
-  }
-
-  @Test
-  public void default_webUrl() {
-    assertThat(underTest.webURL()).isEqualTo("https://bitbucket.org/");
-  }
-
-  @Test
-  public void definitions() {
-    assertThat(BitbucketSettings.definitions()).hasSize(5);
-  }
+    Settings settings = new Settings(new PropertyDefinitions(BitbucketSettings.definitions()));
```

Comment:
You should use the SonarSource coding convention to avoid updating all lines of a file : https://github.com/SonarSource/sonar-developer-toolset

Reply:
I have now reformatted the changed files according to sonar-developer-toolset formatters. Do let me know if it works well for you now.